examples/user_guide/Linked_Brushing.ipynb

```diff
@@ -293,7 +293,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "linked_choropleth = link_selections(choropleth + hist, index_cols=['detailed name'])"
+    "linked_choropleth = link_selections(choropleth + hist, index_cols=[detailed_name])"
```

Comment:
Looking at the definition of `detailed_name`, it was a workaround Bokeh 3.0 change.

`"detailed_name = 'detailed_name' if counties[0].get('detailed_name') else 'detailed name' # detailed name was changed in Bokeh 3.0\n",`

Let us remove `detailed_name` and use `'detailed_name'` wherever needed.